.github/workflows/_run_android_tests.yml

```diff
@@ -11,31 +11,16 @@ jobs:
   build-and-test:
     runs-on: ubuntu-latest
     steps:
-      - name: Setup miniconda
-        uses: conda-incubator/setup-miniconda@v2
-        with:
-          auto-update-conda: true
-          python-version: 3.8
-          activate-environment: build
-
-      - name: Install dependencies
-        run: |
-          conda install -y \
-            cffi=1.15.1 \
-            cmake=3.22.1 \
-            mkl=2022.1.0 \
-            mkl-include=2022.1.0 \
-            ninja=1.10.2 \
-            numpy=1.23.3 \
-            pyyaml=6.0 \
-            requests=2.28.1 \
-            setuptools=65.5.0 \
-            typing_extensions=4.3.0
-
       # [see note: pytorch repo ref]
       - name: Checkout PyTorch
         uses: pytorch/pytorch/.github/actions/checkout-pytorch@master
 
+      - name: Setup miniconda
+        uses: pytorch/test-infra/.github/actions/setup-miniconda@main
+        with:
+          python-version: 3.8
+          environment-file: .github/requirements/conda-env-${{ runner.os }}-${{ runner.arch }}
```

Comment:
It's great to see that the same file can be used for different workflow :)

Reply:
Indeed.